### ISSUE TITLE
Automatically generated replays do not play

### ISSUE THREAD

**Issue by bluecow82** [Jackhod/ACCDirectorAssistant#1]
Thank you for this great app! Only got a chance to play with it for a few minutes today, but I already love it!
It seems that the automatically generated replays don't work. A line pops up saying there is a replay, but clicking on it or the Replay button by it doesn't do anything.

**Comment by Jackhod** [comment]
Yeah I know about that problem :( I'm still searching a way to make them work. Also in the kunos app the replay are a bit broken. If for the next update I haven't found a solution I'll remove that auto generated feature for the moment

**Comment by gersonlizama** [comment]
the ACC Director App on Racedepartment had a working feature of the Replays but instead what it did was allow you to go back 30s and then it would jump back to live. the only problem was you had to sit there and watch the entire replay, you couldnt exit out early
